### ISSUE TITLE
Version 1.0.2 fail with ModuleNotFoundError

### ISSUE THREAD

**Issue by djfjeff** [vivalatech/homeassistant-addons#1]
Hi,

I just installed the latest version (1.0.2), configured my Frame IP address and started the addons. This fail with the following message : 

~~~
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
s6-rc: info: service legacy-services successfully started
Using 192.168.100.29 as the IP of the Samsung Frame
Traceback (most recent call last):
  File "//art.py", line 15, in <module>
    from utils.utils import resize_and_crop_image
ModuleNotFoundError: No module named 'utils'
s6-rc: info: service legacy-services: stopping
s6-rc: info: service legacy-services successfully stopped
s6-rc: info: service legacy-cont-init: stopping
s6-rc: info: service legacy-cont-init successfully stopped
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service fix-attrs successfully stopped
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
~~~

**Comment by djfjeff** [comment]
I am running on HASS OS. HA version 2024.9.1

**Comment by rhaenni** [comment]
sorry about this, has been fixed now in v1.0.3

**Comment by vivalatech** [comment]
sorry about this, has been fixed now in v1.0.3

**Comment by djfjeff** [comment]
Works perfectly after the update, thanks!
